restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
+def _main(filepaths, format, encoding, stream=sys.stdout):
     # Read and lint the file
-    errors = lint_file(filepath, encoding=encoding)
+    file_errors = [lint_file(f, encoding=encoding) for f in filepaths]
+    errors_dicts = []
 
-    # If there were no errors, exit gracefully
-    if not errors:
-        if format == 'json':
-            stream.write('[]')
+    for filename, errors in zip(filepaths, file_errors):
+        if not errors:
+            stream.write('INFO File {} is clean.\n'.format(filename))
         else:
-            stream.write('File was clean.\n')
-        sys.exit(0)
+            if format == 'text':
+                for error in errors:
+                    # e.g. WARNING readme.rst:12 Title underline too short.
+                    stream.write('{} {}:{} {}\n'.format(error.type, error.source, error.line, error.message))
+            elif format == 'json':
+                errors_dicts.extend([{
+                    'line': error.line,
+                    'source': error.source,
+                    'level': error.level,
+                    'type': error.type,
+                    'message': error.message,
+                    'full_message': error.full_message,
+                } for error in errors])
 
-    # Otherwise, output the errors and exit angrily
     if format == 'json':
-        error_dicts = [{
-            'line': error.line,
-            'source': error.source,
-            'level': error.level,
-            'type': error.type,
-            'message': error.message,
-            'full_message': error.full_message,
-        } for error in errors]
-        stream.write(json.dumps(error_dicts))
+        stream.write(json.dumps(errors_dicts))
+
+    if any(file_errors):
+        sys.exit(1)
     else:
-        for error in errors:
-            # WARNING readme.rst:12 Title underline too short.
-            stream.write('%s %s:%s %s\n' % (error.type, error.source, error.line, error.message))
-    sys.exit(1)
+        sys.exit(0)
 
 
 def main():
+    default_encoding = locale.getpreferredencoding()
+
     # Set up options and parse arguments
-    parser = argparse.ArgumentParser(description='Lint a reStructuredText file')
-    parser.add_argument('filepath', type=str, help='File to lint')
-    parser.add_argument('--format', type=str, help='Format of output (e.g. text, json)')
-    parser.add_argument('--encoding', type=str, help='Encoding of the input file (e.g. utf-8)')
+    parser = argparse.ArgumentParser('rst-lint', description='Lint a reStructuredText files')
+    parser.add_argument('filepaths', type=str, nargs='+',
+                        help='File(s) to lint')
+    parser.add_argument('--format', '-f', type=str, default='text',
+                        help='Format of the output (e.g. text, json)')
+    parser.add_argument('--encoding', '-e', type=str, default=default_encoding,
+                        help='Encoding of the input file (e.g. {}, ascii)'.format(default_encoding))
```

Comment:
One additional problem resulting from using `default_encoding` is that this will be a breaking change which means it must be a major release. Whereas the support for multiple filepaths is a non-breaking change.